docs/user/00-25-plug-in-istio-ca.md

```diff
@@ -0,0 +1,44 @@
+# Configure Istio CA with Custom Certificates
+For enhanced security, replace Istio's default self-signed certificates with administrator-provided certificates.
+
+## Default Istio Behavior
+
+By default, Istio generates a self-signed root certificate and uses it to sign certificates for each workload in the service mesh.
+
+Istio follows a standard PKI certificate hierarchy:
+
+- **Root CA**: Self-signed top-level certificate that issues intermediate certificates. It's recommended that this certificate runs on an offline secure machine.
```

Comment:
The Root CA certificate is always a self-signed one. The idea is to run the Root CA outside of the Kubernetes cluster (on an offline secure machine)


```suggestion
- **Root CA**: with a self-signed certificate running outside of the cluster that issues intermediate certificates to the Istio CAs that run in each cluster. It's recommended that this certificate runs on an offline secure machine.
```